### ISSUE TITLE
When using pins, if the tag outputting the bard content is scoped, it causes an error

### ISSUE THREAD

**Issue by kobe-f8** [jacksleight/statamic-bard-texstyle#74]
## Bug Description
When adding scope to your the tag that outputs the bard content, it fails and causes an error.
`Illuminate\Support\Arr::isAssoc(): Argument #1 ($array) must be of type array, string given, called in /Users/macbookair/Documents/Sites/albat/vendor/statamic/cms/src/Support/Arr.php on line 323`

## How to Reproduce
Clean statamic install, enable pins, and add a pin to the bard field. Add a pin to the content.

My antlers code

```
{{ $page_builder scope="block" }}
    {{ %partial src="page_builder/{type}" /}}
{{ /$page_builder }}
```

## Extra Detail
Usually I'm able to share by error using the build in share function but for some reason it doesn't let me so I had to resort to uploading a screenshot of the error, sorry! 😅

![screencapture-albat-test-sandbox-2024-10-15-15_08_42](https://github.com/user-attachments/assets/63fd9885-399a-410f-8b25-d7de76881004)

**Comment by jacksleight** [comment]
I haven't really gotten to the bottom of this, it seems to be some weird behaviour in Statamic (I can reproduce the same issue in a clean install without Texstyle). But I've found a workaround that seems to solve it. Released in v3.3.1.

**Comment by kobe-f8** [comment]
Thanks for looking into it!
